packages/block-library/src/navigation/index.js

```diff
@@ -50,6 +52,14 @@ export const settings = {
 			},
 		],
 	},
+	__experimentalLabel( { ref } ) {
+		const { navigationMenu } = useNavigationMenu( ref );
+		return buildNavigationLabel(
+			navigationMenu.title,
+			1, // This has to be 1 because in this context there is only one Navigation shown.
+			navigationMenu.status
```

Comment:
Accessing property of potentially undefined var will throw error.
```suggestion
			navigationMenu?.status
```